index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Hello</title>
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <h3>Hello World</h3>
```

Comment:
- [ ] [OPTIONAL] It would be better if you use an `<h1>`  instead of  `<h3>` for your header because the H1 tag is important for SEO, accessibility and usability, so ideally, you should have one on each page of your site. [SEO link](https://www.silverdisc.co.uk/blog/2020/11/06/how-important-are-h1-tags-seo)